lib/order.rb

```diff
@@ -0,0 +1,80 @@
+require_relative 'menu'
+require_relative 'text'
+class Order
+    
+    attr_reader :chosen_items, :menu_klass, :final_bill, :text_klass
+    
+    def initialize(menu_klass = Menu.new, text_klass = Text.new)
+        @menu_klass = menu_klass
+        @text_klass = text_klass
+        @chosen_items = Hash.new(0)
+        @final_bill = 0
+        @number_of_dishes = 0
+    end
+    
+    def order_selection(item, quantity)
+        if @menu_klass.view_menu.include?(item) 
+            chosen_items[item] = quantity
+        else
+            raise "please select an item from the menu"
```

Comment:
Style/SignalException: Use `fail` instead of `raise` to signal exceptions. (https://github.com/bbatsov/ruby-style-guide#fail-method)